config/itemTypes.json

```diff
@@ -735,7 +739,7 @@
   "append": true,
   "name": " Mod"
 }, {
-  "id": "/Melee",
+  "id": "/Melee/",
```

Comment:
i'm worried that this is gonna disqualify some that shouldn't be...

Reply:
My bad, I was messing around to try and figure it out before I asked you and forgot to undo the change